.travis.yml

```diff
@@ -12,6 +12,7 @@ script:
   - go generate -run="genmock" -x github.com/google/trillian/...
   - go build -v ./...
   - go test -v ./...
+  - ./integration/map_integration_test.sh
```

Comment:
I've added an `integration_test.sh` which currently just sets some common ENV and calls the `map_integration_test.sh` script.